src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -1553,7 +1553,8 @@ public FormValidation doValidateCredentials(@QueryParameter String roleArn,
                 // This does two things, validates access and secret key are valid and if they have access to device farm.
                 deviceFarm.getProjects();
             } catch (Exception e) {
-               return FormValidation.errorWithMarkup("Invalid Credentials, please refer to troubleshooting " +
+                e.printStackTrace();
```

Comment:
Please add a log like "Error Validating Credentials" so that its better to go through logs and get this information

Reply:
Added logging before printing stacktrace